net/extras/genInventoryFile.py

```diff
@@ -54,6 +54,9 @@ def writeInventory(self, outFd):
             outFd.write(var_line)
             var_line = "netplugin_if={} ".format(connInfo[node]['data'])
             outFd.write(var_line)
+            role = connInfo[node].get('role', 'worker')
```

Comment:
@neelimamukiri: Didn't realize there was a separate playbook that's being used. Changes LGTM in that case.